content/guides/core/registry/delete_registry.md

```diff
@@ -0,0 +1,50 @@
+---
+menu:
+  default:
+    identifier: delete_registry
+    parent: registry
+title: Delete registry
+weight: 8
+---
+
+This page shows how a Team admin or Registry admin can delete a custom registry.  A [core registry]({{< relref "/guides/core/registry/registry_types#core-registry" >}}) cannot be deleted.
+
+- A Team admin can delete any custom registry in the organization.
+- A Registry admin can delete a custom registry that they created.
+
+Deleting a registry also deletes collections that belong to that registry, but does not delete artifacts linked to the registry.  Such an artifact remains in the original project that the artifact was logged to.
```

Comment:
Nit for later follow up: A few double spaces between sentences crept into the page.